cmd/serverless-init/log/log.go

```diff
@@ -44,14 +48,17 @@ func CreateConfig(metadata *metadata.Metadata) *Config {
 		channel:      make(chan *logConfig.ChannelMessage),
 		source:       source,
 		loggerName:   loggerName,
+		isEnabled:    isEnabled(os.Getenv(logEnabledEnvVar)),
```

Comment:
I think in the long run it make sense to read this using the existing conf utilities. We can leave it as is for now though.